programmers/난이도별/level01.문자열_다루기_기본/yeeun.js

```diff
@@ -0,0 +1,11 @@
+function solution(s) {
+    var answer = true;
+    var rglarExp = {containsNumber: /^[0-9]+$/} 
+    //다른 간단한 정규식 : /^\d{6}$|^\d{4}$/
+    if(s.length===4||s.length===6){
+        answer = rglarExp.containsNumber.test(s);
+    }else{
+        answer=false;
+    }
+    return answer;
```

Comment:
```suggestion
    return [4, 6].includes(s.length) && /^[0-9]+$/.test(s)
```
이렇게 표현할 수 있습니다!

혹은
```js
    return [4, 6].includes(s.length) && /^\d+$/.test(s)
```
```js
    return /^([0-9]{4}|[0-9]{6})$/.test(s);
```
```js
    return /^(\d{4}|\d{6})$/.test(s);
```

이렇게 표현할 수 있습니다. 이미 주석에 달아주셨네요 ㅎㅎ